src/main/kotlin/kweb/WebBrowser.kt

```diff
@@ -135,6 +135,9 @@ class WebBrowser(val sessionId: String, val httpRequestInfo: HttpRequestInfo, va
     }
 
     suspend fun callJsFunctionWithResult(jsBody: String, vararg args: JsonElement): JsonElement {
+        require(kweb.isCatchingOutbound() == null) {
+            "You can not read the DOM inside a batched code block"
```

Comment:
"You can not use callJsFunctionWithResult() inside a batched code block like a render or event handler"